source/max-staleness/max-staleness.md

```diff
@@ -0,0 +1,520 @@
+> '10', '25 seconds'\], \['80', '30', 'P responds', '70', '5', '80', '30', '25 seconds'\]
+>
+> \]
+>
+> print Table(data).create_table()
+
+## Estimating Staleness: Example With No Primary
+
+Consider a replica set with secondaries S1 and S2, and no primary. S2 lags 15 seconds *farther* behind S1 and has not
+yet caught up. The client has heartbeatFrequencyMS set to 10 seconds.
+
+When the client checks the two secondaries, S1's lastWriteDate is 20 and S2's lastWriteDate is 5.
+
+Because S1 is the secondary with the maximum lastWriteDate, "SMax", its staleness estimate equals heartbeatFrequencyMS:
+
+> SMax.lastWriteDate - S.lastWriteDate + heartbeatFrequencyMS = 20 - 20 + 10 = 10
+
+(Since max staleness must be at least heartbeatFrequencyMS + idleWritePeriodMS, S1 is eligible for reads no matter
+what.)
+
+S2's staleness estimate is:
+
+```
+SMax.lastWriteDate - S.lastWriteDate + heartbeatFrequencyMS
+= 20 - 5 + 10
+= 25
+```
+
+## Estimating Staleness: Example of Worst-Case Accuracy With Idle Replica Set
+
+Consider a primary P and a secondary S, and a client with heartbeatFrequencyMS set to 500 ms. There is no clock skew.
+(Previous examples show that skew has no effect.)
+
+The primary has been idle for 10 seconds and writes a no-op to the oplog at time 50 (meaning 50 seconds past midnight),
+and again at time 60.
+
+Before the secondary can replicate the no-op at time 60, the client checks both servers. The primary reports its
+lastWriteDate is 60, the secondary reports 50.
+
+The client estimates S's staleness as:
+
+```
+(S.lastUpdateTime - S.lastWriteDate) - (P.lastUpdateTime - P.lastWriteDate) + heartbeatFrequencyMS
+= (60 - 50) - (60 - 60) + 0.5
+= 10.5
+```
+
+The same story as a table:
+
+| Clock | Event                 | S.lastUpdateTime | S.lastWriteDate | P.lastUpdateTime | P.lastWriteDate | S staleness |
+| ----- | --------------------- | ---------------- | --------------- | ---------------- | --------------- | ----------- |
+| 50    | Idle write            | 50               |                 | 50               |                 |             |
+| 60    | Idle write begins     | 60               |                 | 50               |                 |             |
+| 60    | Client checks P and S | 60               | 60              | 50               | 60              | 10.5        |
+| 60    | Idle write completes  | 60               |                 | 60               |                 |             |
+
+> from table import Table
+>
+> data = \[\
+> \['Clock', 'Event', 'S.lastUpdateTime', 'S.lastWriteDate',\
+> 'P.lastUpdateTime', 'P.lastWriteDate', 'S
+> staleness'\],
+>
+> \['50', 'Idle write', '50', '', '50', '', ''\], \['60', 'Idle write begins', '60', '', '50', '', ''\], \['60', 'Client
+> checks P and S', '60', '60', '50', '60', '10.5'\], \['60', 'Idle write completes', '60', '', '60', '', ''\],
+>
+> \]
+>
+> print Table(data).create_table()
+
```

Comment:
This was a comment in the original code, are we intentionally keep it as a quote?

https://github.com/mongodb/specifications/blob/39add29e0a3b6721688845874383878b90811a12/source/max-staleness/max-staleness.rst?plain=1#L344-L357

```suggestion
```